plonetheme/barceloneta/theme/less/behaviors.plone.less

```diff
@@ -1,11 +1,10 @@
 //*// BEHAVIORS //*//
 
 .newsImageContainer {
-	float:right;
-	margin: 0 0 @plone-padding-base-vertical @plone-padding-base-vertical;
-	border-radius: @plone-border-radius-base;
-	box-shadow: 0 1px 2px rgba(0,0,0,.17);
-	overflow: hidden;
+	float: right;
+    margin: 0 0 6px 6px;
+    border-radius: 4px;
+    overflow: hidden;
```

Comment:
Did you intentionally hard hardcode these sizes? Do these sizes differ from the `@plone-*` variables that were there? If not, they should stay the same.

Note: I don't know much about the Plone less/css code, I am just wondering.